README.md

```diff
@@ -103,6 +103,45 @@ npm install validimir
 
   Assert each of value - no matter whether it's an array or object - passes `fn` with should be a function returned by validimir or an api compatible module.
 
+### .custom(fn)
+
+  Add custom check.
+
+  This function adds a new check function `fn`.
+
+  A check function in validimir is a function that gets a value, tries to validate it and returns either nothing (or any falsy value) on success or an error object on failure. There are no strict requirements with regard to the error object, but to be consistent with other checks in validimir, it should have the following properties:
+  - `value`: The value that didn't pass the validation
+  - `operator`: The name of the operator that failed
+  - `message`: Descriptive error messsage
+  - `expected`: The expected value, if any, depending on the check
+
+  Example:
+
+```js
+var v = require('validimir');
+var isIP = require('validator').isIP
```

Comment:
@juliangruber Ok, so I added missing semicolons to the example in 383c034 and a couple more that I think you missed in 873f98b